### ISSUE TITLE
Boldface problem

### ISSUE THREAD

**Issue by cz302** [veusz/veusz#511]
Hej,

I have been using Veusz for more than a decade. Thank you very much for the great work!

However I noticed in recent version of Veusz, the boldface button for labels is no longer functioning.

Best,

Chao

**Comment by cz302** [comment]
The issue is about the decimal on MacOS. If it uses the default "," in the Swedish system, then the boldface function becomes inactivated. By changing the decimal to ".", the boldface problem dissolves.

**Comment by jeremysanders** [comment]
So this is still a problem if using the European number format? Weird. If so, it's probably worth keeping open and I can try to take a look when I get time.

**Comment by cz302** [comment]
Yes, it is a weird issue. Just by chance, I notice today that they are correlated.
